pkgs/by-name/th/the-dark-mod/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  symlinkJoin,
+  writeShellApplication,
+  makeDesktopItem,
+  iconConvTools,
+
+  addDriverRunpath,
+  libGL,
+  xorg,
+  alsa-lib,
+
+  the-dark-mod-unwrapped,
+  the-dark-mod-assets,
+}:
+let
+  libPath = lib.makeLibraryPath [
+    # GLFW
+    addDriverRunpath.driverLink
+    libGL
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXrandr
+    xorg.libXxf86vm
+
+    # OpenAL
+    alsa-lib
+  ];
+
+  runScript = writeShellApplication {
+    name = "the-dark-mod";
+    runtimeInputs = [ xorg.lndir ];
+
+    text = ''
+      gameDir=''${XDG_DATA_HOME:-$HOME/.local/share}/darkmod
+      mkdir -p "$gameDir"
+      lndir -silent ${the-dark-mod-assets} "$gameDir"
```

Comment:
What if the assets are updated (or a file gets removed): do stale links linger?

Reply:
I think so? We might need to purge old links then